apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -40,10 +40,10 @@ export const commands = {
   getThisSprite(which, extraArgs) {
     if (extraArgs) {
       if (which === 'this') {
-        return extraArgs.sprite;
```

Comment:
Just to double check - the extraArgs.sprite and extraArgs.targets are ids?

Reply:
Yeah, set here:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/p5lab/spritelab/coreLibrary.js#L224